events/wc_cenarion_circle_events.txt

```diff
@@ -205,6 +208,25 @@ character_event = {
 		move_character = FROM
 	}
 }
+character_event = {
+	id = WCCC.23
+
```

Comment:
Has no localisation, picture and borders.

Reply:
fixed